src/styles/globals.css

```diff
@@ -63,6 +63,17 @@ html {
   --blue-800: #1e40af;
   --blue-900: #1e3a8a;
 
+  --orange-50: #fff7ed;
```

Comment:
These are unrelated changes (we had a warning icon in the DataBlock component originally, I left it out this time), but we'll need the orange colors anyways, so I left these lines here.